swagger/v2/swagger.yaml

```diff
@@ -139,6 +139,12 @@ paths:
           description: Filters files by path. Requires query parameter project_identifier.
           required: false
           type: string
+        - name: pagination
```

Comment:
I guess this was missing from some previous PR and now added here?

Reply:
Rebase does sometimes strange things... This branch was rebased so many times that have no idea with this one either why it's here, but decided to keep it so it won't disappear.

Reply:
Should be in next PR... will delete this